### ISSUE TITLE
Research: CI/CD pipeline

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#4]
## User story
1. As a team
2. We need an automated CI/CD pipeline
3. In order to ensure high code quality

## Acceptance criteria
* Research how to use [GitHub Actions](https://docs.github.com/en/actions/using-workflows/about-workflows)
* Create a simple CI/CD pipeline that runs on every commit
* Research python linters and try to integrate one into the CI pipeline
* Research how automated python unittests can be integrated into the GitHub CI pipeline and do so if possible

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
